src/librustdoc/formats/cache.rs

```diff
@@ -472,7 +472,7 @@ impl DocFolder for Cache {
         });
 
         if pushed {
-            self.stack.pop().expect("stack already empty");
+            let _ = self.stack.pop().expect("stack already empty");
```

Comment:
I think lines like this are a good example of why `String` and `Vec<T>` aren't `must_use`.

Reply:
These cases are rare in Rustc codebase.

Reply:
I agree with @scottmcm here. If it's on String, then it should be on Vec too, and if it's on Vec, then it should also be on every container. At which point you'd have to somewhat argue why it's not on basically everything that's not Copy.

Reply:
With all respects, I just want some actions on resolving #75742 . If the team doesn't like this approach,
what about marking `String::from`, `str::into`, `fmt::format` with `must_use` attr?